community/planning/features.md

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
The copyright headers for these files is incorrect; I'm in the process of updating the existing ones throughout the website. The headers should be below the title heading (e.g. `# Feature Planning`) so the Jekyll plugin that infers the page title works properly, and they should be the following string:

```
<!--
  Copyright 2018-2020 Cargill Incorporated
  Licensed under Creative Commons Attribution 4.0 International License
  https://creativecommons.org/licenses/by/4.0/
-->
```

Reply:
Updated